holmes/core/supabase_dal.py

```
@@ -606,29 +621,184 @@ def get_skill_content(self, skill_id: str) -> Optional[RobustaSkillInstruction]:
+                # different cluster cannot suppress an applicable one.
+                if clusters is not None and self.cluster not in clusters:
+                    continue
+                # Validate per row. id and title are required on the model, so a row with a
+                # null runbook_id or subject_name raises -- and if that reached the outer
+                # handler the user would silently lose EVERY personal skill, not just the
+                # malformed one. Skip the bad row instead.
+                try:
+                    instructions.append(
+                        RobustaSkillInstruction(id=id, symptom=symptom, title=title)
+                    )
+                except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)